apps/web/pages/reschedule/[uid].tsx

```diff
@@ -136,6 +136,10 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
 
   destinationUrl.set("rescheduleUid", seatReferenceUid || bookingUid);
 
+  // TODO: I think we should just forward all the query params here including coep flag
+  if (coepFlag) {
```

Comment:
Directly opening /reschedule/[uid] endpoint seems to lose the flag on redirect, so we explicitly add it.